js/huobi.js

```diff
@@ -2954,12 +2954,6 @@ module.exports = class huobi extends Exchange {
                 request['account-id'] = accountId;
                 method = 'spotPrivateGetV1AccountAccountsAccountIdBalance';
             }
-        } else if (margin) {
-            if (isolated) {
-                method = 'spotPrivateGetV1MarginAccountsBalance';
-            } else {
-                method = 'spotPrivateGetV1CrossMarginAccountsBalance';
-            }
```

Comment:
these endpoints were just duplicated, they should have been in above `if (spot || margin)` clause, like I've edited.